src/slave/containerizer/docker.cpp

```diff
@@ -1303,11 +1303,14 @@ void DockerContainerizerProcess::destroy(
     container->termination.set(termination);
 
     containers_.erase(containerId);
+
+    remove(container->name(), None());
```

Comment:
Here (and in a number of places throughout this file), we're removing stopped Docker containers. Billing accrues for every provisioned container in Joyent's container-native infrastructure, so this step eliminates the need for garbage collection of stopped containers.